helpers/securewebhook/securewebhook.go

```diff
@@ -27,3 +38,25 @@ func GetRequestBody(s *Settings) ([]byte, error) {
 	}
 	return b, nil
 }
+
+// VerifySignature ...
```

Comment:
could you explain what `VerifySignature` does?

Reply:
added